modules/serialize/src/aad_factory.ts

```diff
@@ -44,7 +44,7 @@ export function aadFactory (fromUtf8: (input: string) => Uint8Array) {
           ? ContentAADString.FINAL_FRAME_STRING_ID
           : ContentAADString.FRAME_STRING_ID
       default:
-        throw new Error('bad')
+        throw new Error('Unrecognized context type')
```

Comment:
```suggestion
        throw new Error('Unrecognized content type')
```

Reply:
Also, should this be a particular type of error?

Reply:
I have not gotten to "types of errors" _yet_.  Error codes are a thing :(